packages/@okta/vuepress-site/docs/guides/refresh-tokens/refresh-token-rotation/index.md

```diff
@@ -0,0 +1,62 @@
+
+## Enable refresh token rotation
+
+Rotating refresh token behavior is the default for SPAs. When you create a new SPA, or when you update an existing SPA, and select **Refresh Token** as the allowed grant type, rotating the refresh token is set as the default.
+
+To update existing OpenID Connect applications to use refresh token rotation:
+
+1. From the Developer Console, click **Applications** and then select the application that you want to configure.
+2. Select the **General** tab and click **Edit**.
+3. In the **Allowed grant type** section, select **Refresh Token**.
+4. In the **REFRESH TOKEN** section that appears, select **Rotate token after every use**.
+5. Make any adjustments to the number of seconds for the **Grace period for token rotation**. You can change the value to any number between 0 and 60 seconds. After the refresh token is rotated, the previous token remains valid for this amount of time to allow clients to get the new token.
+
+When you create a new native or web app and want to use refresh token rotation:
+
+1. From the Developer Console, click **Applications** and then **Add Application**.
+2. While walking through the app creation wizard, select **Refresh Token** in the **Allowed grant type** section.
+3. After you click **Done**, select the **General** tab and click **Edit**.
+4. In the **REFRESH TOKEN** section, select **Rotate token after every use**.
+5. Make any adjustments to the number of seconds for the **Grace period for token rotation**. You can change the value to any number between 0 and 60 seconds. After the refresh token is rotated, the previous token remains valid for this amount of time to allow clients to get the new token.
+
+### Refresh token rotation properties
+
+After you enable refresh token rotation, the `refresh_token` property appears within `settings.oauthClient` for your app. The `refresh_token` property is an array that contains the `rotation_type` and `leeway` parameters. Accepted values for `rotation_type` are `ROTATE` or `STATIC`. The accepted value for `leeway` is any number between 0 and 60.
+
+```bash
+"refresh_token": {
+                "rotation_type": "ROTATE",
+                "leeway": 30
+                 }
+```
+
+See [somewhere in the Apps API-LARS](/docs/reference/api/apps/).
+
+## Refresh token lifetime
+
+Refresh token lifetimes are managed through the [Authorization Server access policy](/docs/reference/api/authorization-servers/#actions-object). When you use a refresh token with a SPA, make sure that you keep a short refresh token lifetime for better security.
```

Comment:
Should we also link the product link here as well for UI configuration? https://help.okta.com/en/prev/Content/Topics/Security/API_Access.htm?cshid=create-access-policies#create-access-policies

Reply:
Should we mention the default lifetimes for Org AS as well?

Reply:
> Should we also link the product link here as well for UI configuration? https://help.okta.com/en/prev/Content/Topics/Security/API_Access.htm?cshid=create-access-policies#create-access-policies

@venkatviswanathan-okta - We have this now. I'll link it to that: https://developer.okta.com/docs/guides/configure-access-policy/overview/